data/abdulahad-07.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Abdul Ahad",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source | Full-stack Developer",
+  "avatar": "https://github.com/abdulahad-07.png",
```

Comment:
Avatar key is no longer used since we are using your GitHub profile as default avatar. So Please remove this from your file. 
```suggestion
```

**Tip:** To make the requested changes, you can use the <kbd>Commit suggestions</kbd> button above.